docs/essentials/collections/whentoUseGenericCollections.md

```diff
@@ -0,0 +1,45 @@
+Using generic collections is generally recommended, because you gain the immediate benefit of type safety without having to derive from a base collection type and implement type-specific members. Generic collection types also generally perform better than the corresponding nongeneric collection types (and better than types that are derived from nongeneric base collection types) when the collection elements are value types, because with generics there is no need to box the elements. 
+
+You should use the generic collection classes in the [System.Collections.Concurrent](https://dotnet.github.io/api/System.Collections.Concurrent.html) namespace when multiple threads might be adding or removing items from the collection concurrently.
+
+The following generic types correspond to existing collection types: 
+
+*   [List<T>](https://dotnet.github.io/api/System.Collections.Generic.List%601.html) is the generic class that corresponds to [ArrayList](https://dotnet.github.io/api/System.Collections.ArrayList.html).
+
+*   [Dictionary<TKey, TValue>](https://dotnet.github.io/api/System.Collections.Generic.Dictionary%602.html) and [ConcurrentDictionary<TKey, TValue>](https://dotnet.github.io/api/System.Collections.Concurrent.ConcurrentDictionary%602.html) are the generic classes that correspond to [Hashtable](https://dotnet.github.io/api/System.Collections.Hashtable.html). 
+
+*   [Collection<T>](https://dotnet.github.io/api/System.Collections.ObjectModel.Collection%601.html) is the generic class that corresponds to [CollectionBase](https://dotnet.github.io/api/System.Collections.CollectionBase.html). **Collection<T>** can be used as a base class, but unlike **CollectionBase**, it is not abstract. This makes it much easier to use.
```

Comment:
I think you need to prefix special characters with backslash to make them render correctly. Otherwise, the generic parameter doesn't render as in [Collection\<T\>](https://dotnet.github.io/api/System.Collections.ObjectModel.Collection%601.html)